tests/YotiClientTest.php

```diff
@@ -172,6 +172,24 @@ public function testCanUsePHPAsSdkIdentifier()
         $this->assertEquals($property->getValue($yotiClientObj), $expectedValue);
     }
 
+    public function testRequestHeadersShouldIncludeSDKVersion()
+    {
+        $yotiClientObj  = new YotiClient(
+            SDK_ID,
+            $this->pem,
+            YotiClient::DEFAULT_CONNECT_API,
+            'PHP'
+        );
+        $requestHeaders = $this->invokeMethod(
```

Comment:
Same consideration as for https://github.com/getyoti/yoti-php-sdk/pull/40 applies.
If possible, try to avoid usage of reflection to test methods.
You can try to mock whatever is performing the http request and verify the values of the headers in the request.

Reply:
This has been addressed in [SDK-658](https://github.com/getyoti/yoti-php-sdk/blob/SDK-658/tests/Http/CurlRequestHandlerTest.php#L80), so will be reflected here once merged.